assets/migrations/mysql/005_drop_indices.sql

```diff
@@ -0,0 +1,28 @@
+--  +goose Up
+--   This migration updates the primary key of the tuple table
+
+ ALTER TABLE tuple
```

Comment:
It doesn't appear that this is a locking operation. I was able to continue reading/writing concurrently while this was doing it's work.